### DISCUSSION TITLE
Getting the correct number of github contributors via the API

### DISCUSSION THREAD

**Discussion by novica** [community/community#24551] [Apps, API and Webhooks]
<p>I am trying to fetch the contributors for the <code>bitcoin</code> repository via the api.</p>
<p>When I use <code>GET /repos/:owner/:repo/contributors</code> I get 362 elements back</p>
<p>I assumed that there might be anonimous contributors so used <code>?anon=1</code> and then got 860.</p>
<p>But I can’t seem be abel to get the number listed on the repository page – 699 contrubutors.</p>
<p>What I am doing wrong?</p>

**Comment by ilia-cy** [comment, marked as answer]
<p><a href="https://github.community/t/api-github-list-contributors/13520">https://github.community/t5/GitHub-API-Development-and/API-Github-list-contributors/td-p/9631</a></p>
